File: app/src/main/java/com/innovation/rain/feature/agentlogin/view/AgentLoginView.java
package com.innovation.rain.feature.agentlogin.view;


import com.innovation.rain.app.base.presenter.BasePresenterView;

public interface AgentLoginView extends BasePresenterView {
    void notifyEmptyPw();

    void notifyEmptyUserId();

    void enableButtonSignIn(boolean allowEnableSignInButton);
}

File: app/src/main/java/com/innovation/rain/feature/agentlogin/view/AgentLoginView.kt
package com.innovation.rain.feature.agentlogin.view


import com.sf0404.core.application.base.presenter.BaseView

interface AgentLoginView : BaseView {
    fun enableButtonSignIn(allowEnableSignInButton: Boolean)

    fun gotoWelcomeMenuScreen()

    fun showAlreadyLoginError()

    fun showAgentNotFoundError()
}
